java/test/jmri/jmrix/ieee802154/xbee/configurexml/ConnectionConfigXmlTest.java

```diff
@@ -1,20 +1,68 @@
 package jmri.jmrix.ieee802154.xbee.configurexml;
 
+import com.digi.xbee.api.XBeeDevice;
+import jmri.jmrix.ieee802154.xbee.XBeeNode;
+import jmri.jmrix.ieee802154.xbee.XBeeNodeManager;
+import jmri.jmrix.ieee802154.xbee.XBeeTrafficController;
 import jmri.util.JUnitUtil;
 
+import jmri.util.ThreadingUtil;
+import org.jdom2.Element;
+import org.junit.Assume;
 import org.junit.jupiter.api.*;
 
 import jmri.jmrix.ieee802154.xbee.ConnectionConfig;
+import org.mockito.Mockito;
+
+import javax.swing.*;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.catchThrowable;
 
 /**
  * ConnectionConfigXmlTest.java
  *
  * Test for the ConnectionConfigXml class
  *
- * @author   Paul Bender  Copyright (C) 2016
- */
+ * @author   Paul Bender  Copyright (C) 2016*/
```

Comment:
No need to change, but this change looks like a misstake. That `*/` is on the same line as the copyright.

Reply:
yeah, that was probably caused by me hitting the touchpad on my laptop.  
Fixed.